xds/go13.go

```diff
@@ -0,0 +1,25 @@
+// +build go1.13
```

Comment:
Nit: filename should be `go113.go` to match our more commonly-used convention (there is one `go12.go`).

Reply:
Done.